ts/store/reducers/content.ts

```diff
@@ -81,19 +95,26 @@ export const contextualHelpDataSelector = (
   state: GlobalState
 ): pot.Pot<ContextualHelp, Error> => state.content.contextualHelp;
 
+export const idpsSelector = createSelector(
+  contentSelector,
+  (content: ContentState): ReadonlyArray<SpidIdp> =>
+    isReady(content.idps) ? content.idps.value.items : idps
```

Comment:
I think that, following the same approach of the bonuses list, we should avoid having a local fallback that may not be in sync with the remote endpoint and handle the loading / error before to access to the IDPs screen selection. What you think?

Reply:
With @Undermaken we decided to keep a local fallback when no remote idps data is available to avoid blocking the user's login flow?

Reply:
Since SPID IDP data and login API are hosted in two differente systems (CDN and IO-Backend) if CDN doesn't work it doesn't mean IO-backend is not working too. In that case we are denying login.

So I think a fallback could be a rough solution. But we could think about other, and better, strategies.
Another approach could be making a retry logic (maybe this PR already implements it - I don't know) where the user can ask to reload the data. 

Most important is monitoring the failures with MP

Reply:
@Undermaken @fabriziofff yeah, I've added the mixpanel track for load of idps data and contextualHelp data on 61a303a so we're able to track rates for these informations too

Reply:
I understand the need to guarantee the login in any case, my only fear is that it becomes an entity that must be maintained in sync with the remote (without automation) otherwise in case of malfunctions unexpected behavior can occur as in the case of the fallback of the bonus list.
If we decided to keep a local copy of the Idps, I suggest to move into another file because `ts/store/reducers/__mock__/idps.ts` it makes me think that they are mock data, created for test purpose, that I can modify without much hesitation to test any specific case.

Reply:
> Another approach could be making a retry logic (maybe this PR already implements it - I don't know) where the user can ask to reload the data.

What if the downtime of the remote source is too long? User may be locked for a long time. Anyway this PR doesn't include a retry logic, but I can add it using the fallback in some specifical error cases

Reply:
> I suggest to move into another file because `ts/store/reducers/__mock__/idps.ts` it makes me think that they are mock data, created for test purpose, that I can modify without much hesitation to test any specific case.

Yep! you're right it is confusing